terraform/environments/core-shared-services/ad-fixngo.tf

```diff
@@ -0,0 +1,786 @@
+            protocol    = "UDP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          kerberos-tcp-88 = {
+            port        = 88
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          ntp-udp-123 = {
+            port        = 123
+            protocol    = "UDP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          rpc-tcp-135 = {
+            port        = 135
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          netbios-tcp-139 = {
+            port        = 139
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          ldap-udp-389 = {
+            port        = 389
+            protocol    = "UDP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          ldap-tcp-389 = {
+            port        = 389
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          smb-tcp-445 = {
+            port        = 445
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          kerberos-udp-464 = {
+            port        = 464
+            protocol    = "UDP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          kerberos-tcp-464 = {
+            port        = 464
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          ldaps-tcp-636 = {
+            port        = 636
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          ldap-global-catalog-tcp-3268 = {
+            from_port   = 3268
+            to_port     = 3269
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          ldaps-tcp-3269 = {
+            port        = 3269
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          rdp-tcp-3389 = {
+            port     = 3389
+            protocol = "TCP"
+            cidr_blocks = flatten([
+              module.ad_fixngo_ip_addresses.mp_cidr.hmpps-preproduction,
+              module.ad_fixngo_ip_addresses.mp_cidr.hmpps-production,
+            ])
+          }
+          winrm-tcp-5985-5986 = {
+            from_port = 5985
+            to_port   = 5986
+            protocol  = "TCP"
+            cidr_blocks = flatten([
+              module.ad_fixngo_ip_addresses.mp_cidr.hmpps-preproduction,
+              module.ad_fixngo_ip_addresses.mp_cidr.hmpps-production,
+            ])
+          }
+          adws-tcp-9389 = {
+            port        = 9389
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+          rpc-tcp-dynamic2 = {
+            from_port   = 49152
+            to_port     = 65535
+            protocol    = "TCP"
+            cidr_blocks = ["10.0.0.0/8"]
+          }
+        }
+        egress = {
+          all = {
+            port        = 0
+            protocol    = -1
```

Comment:
this is a bit open

Reply:
Rules here are mirroring the domain controller NSGs in Azure.  The Domain Controller ports allow 10/8 - so any end user device connected onto our network can talk to the DC, and also the DOM1 DCs can reach as well for the forest trust.  Outbound we allow everything to keep things simple - in fact, this is what we have for all our EC2s in all our applications.  Assumption is there is another layer of control, e.g. firewall, gating the outbound access to internet.

Reply:
I think the firewall rules are a bit open actually, but let me check.

Reply:
Just checked and they are not as open as these SG :) I think there is enough balance there though.